bot/commands/command_logic/pomo.py

```diff
@@ -49,13 +54,18 @@ def __init__(self, username: str, ctx: Message, work_minutes: int, break_minutes
         self.total_sessions = sessions
         self.sessions_remaining = sessions
         self.topic = topic
+
+        self.__ctx = ctx
```

Comment:
hmm not sure of risks of storing the ctx here for the duration of the timer. think it's probably safer to pass it in whenever needed....

Reply:
if you are only using it to send a message

Reply:
ctx is stored so that the timer can send notifications using the `ctx.channel.send` method. Only other way to possibly do this would be to past in anon functions to be saved on the timer to replace the send function